xray/manager.go

```diff
@@ -132,6 +132,36 @@ func (sm *XrayServicesManager) GetScanGraphResults(scanID string, includeVulnera
 	return scanService.GetScanGraphResults(scanID, includeVulnerabilities, includeLicenses)
 }
 
+// BuildScan will send Xray the given build for scan
+// Returns a string represents the scan ID.
+func (sm *XrayServicesManager) BuildScan(params services.XrayBuildParams) (string, error) {
+	buildScanService := services.NewBuildScanService(sm.client)
+	buildScanService.XrayDetails = sm.config.GetServiceDetails()
+	return buildScanService.BuildScan(params)
+}
+
+// GetBuildScanResults returns an Xray build scan output of the requested build scan.
+// The scanId input should be received from BuildScan request.
+func (sm *XrayServicesManager) GetBuildScanResults(params services.XrayBuildParams) (*services.BuildScanResponse, error) {
+	buildScanService := services.NewBuildScanService(sm.client)
+	buildScanService.XrayDetails = sm.config.GetServiceDetails()
+	return buildScanService.GetBuildScanResults(params)
+}
+
+// BuildSummary will get the build summary from Xray
```

Comment:
BuildSummary will get the build summary from Xray
-->
BuildSummary returns the a summary of build scan which had been previously performed.